### ISSUE TITLE
Combine into a single branch?

### ISSUE THREAD

**Issue by shawntabrizi** [stiiifff/substrate-poe-tutorial#5]
I foresee maintenance of this repository to be more difficult if each step is in its own branch.

Rather we just keep everything in separate folders in a single branch might be easier?

IDK, I don't really have a preference, just wanted to put it out there.

We could probably remove "level-2" and just use "master" branch

**Comment by stiiifff** [comment]
Agreed. However, assuming that we would then have a subfolder with a complete copy of the source for each step, this would imply something a bit more complex on the developer side (plus consuming way more disk space ... I'm at ~8GB for dev / release build). Current idea is to do any fix on the dev branch, then cherrypick & back port them on level-x branches.
"level-2" branch is the starting point for level 2 step (= end of level 1) so not equivalent to master which contains the fully implement sample.

**Comment by shawntabrizi** [comment]
I think for the actual user, they will only have a single project open, and they will be starting on their own (not cloning this repo).

I think this repo comes handy when we want to point to finished source code which we know works. Really though, leave it to your judgement here.
